client/src/components/Signup.js

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+
+const apiUrl = 'http://localhost:5000/api';
+
+class Signup extends Component {
+    constructor() {
+        super();
+        this.state = {
+            username: 'testing2',
+            password: '123445',
+            department: 'accounting',
+        };
+    }
+
+    handleChange = event => {
+        const { name, value } = event.target;
+
+        this.setState({ [name]: value });
+    };
+
+    handleSubmit = event => {
+        event.preventDefault();
+
+        const endPoint = `${apiUrl}/register`;
+
+        axios
+            .post(`${endPoint}`, this.state)
```

Comment:
Since you're not adding anything onto the endPoint string, you could pass it alone here:
```javascript
.post(endPoint, this.state)
```